.github/workflows/release-node-bindings.yml

```diff
@@ -83,17 +83,22 @@ jobs:
         uses: cargo-bins/cargo-binstall@main
 
       - name: Install cargo-zigbuild
-        run: cargo binstall cargo-zigbuild -y
+        run: cargo binstall cargo-zigbuild@0.20.0 -y
 
       - name: Install cargo-xwin (Windows on Linux only)
         if: contains(matrix.settings.target, 'windows')
         run: cargo binstall cargo-xwin -y
 
-      - name: Build (Mac and Linux)
-        if: contains(matrix.settings.target, 'windows') == false
+      - name: Build Linux
+        if: contains(matrix.settings.target, 'linux')
         run: yarn build --zig --release --target ${{ matrix.settings.target }}
         working-directory: bindings/node
 
+      - name: Build Mac
+        if: contains(matrix.settings.target, 'apple')
+        run: yarn build --release --target ${{ matrix.settings.target }}
+        working-directory: bindings/node
+
```

Comment:
cross compiling mac on linux using zig-build seems to no longer be that straightforward